src/program/snabbmark/snabbmark.lua

```diff
@@ -45,18 +45,18 @@ function basic1 (npackets)
    config.app(c, "Source", basic_apps.Source)
    config.app(c, "Tee", basic_apps.Tee)
    config.app(c, "Sink", basic_apps.Sink)
-   config.link(c, "Source.tx -> Tee.rx")
-   config.link(c, "Tee.tx1 -> Sink.rx1")
-   config.link(c, "Tee.tx2 -> Sink.rx2")
+   config.link(c, "Source.output -> Tee.input")
+   config.link(c, "Tee.output1 -> Sink.input1")
+   config.link(c, "Tee.output2 -> Sink.input2")
    engine.configure(c)
    local start = C.get_monotonic_time()
    timer.activate(timer.new("null", function () end, 1e6, 'repeating'))
-   while link.stats(engine.app_table.Source.output.tx).txpackets < npackets do
+   while link.stats(engine.app_table.Source.output.output).output_packets < npackets do
```

Comment:
So here is a potential mistake in translation that could affect benchmark results (although not of the l2fwd benchmark). I changed “packets transmitted onto the link” to ”packets read off the link” because I felt it was correct in this instance, and further instances below. E.g. I think its “cheating” to stop when the Source has emitted N packets, as opposed to when these N packets have actually been processed.

Opinions?